### PR TITLE
fix: action bar buttons on mobile

### PR DESCRIPTION
**Pull request by anoblet**
https://app.clickup.com/t/86azp8r7v

### CODE REVIEW

**Review by paulkirspuu — APPROVED** [review]
Looks to be working in my WPS.

**Review by pawelkmpt — APPROVED** [review]

File: includes/class-sensei-quiz.php [hunk 1]
```diff
@@ -1315,38 +1315,73 @@ public static function action_buttons() {
...
+					<vaadin-button
+						type="submit"
+						name="quiz_complete"
+						class="quiz-submit complete"
+						theme="primary"
+					>
+						<?php esc_html_e( 'Submit', 'sensei-lms' ); ?>
+					</vaadin-button>
+				</div>
+				<div class="desktop">
```
Comment (lkraav) [inline includes/class-sensei-quiz.php:1374]: Not a fan of these invented device category classnames. Something that matches actual library media queries, "mq-small" or similar would help make the connection better.

Alternatively, library would refactor itself appropriately.

Comment (anoblet) [reply includes/class-sensei-quiz.php:1374]: I agree with this. Should I submit a PR to fix?

Comment (pawelkmpt) [reply includes/class-sensei-quiz.php:1374]: you can file a followup @anoblet

**Pull request merged**